src/components/base/CopyToClipboard.js

```diff
@@ -1,8 +1,9 @@
 // @flow
+import { STORYBOOK_ENV } from 'config/constants'
 
 let clipboard = null
 
-if (!process.env.STORYBOOK_ENV) {
+if (!STORYBOOK_ENV) {
```

Comment:
will not work with how storybook webpack handle that (that's why i changed it this morning)